src/main/java/Task.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Task class
+ * Stores information on task.
+ */
+public class Task {
+    // ATTRIBUTES
+    protected String description;
+    protected boolean isDone;
+
+    // METHODS
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    protected String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    /**
+     * Return status and description of task
+     */
```

Comment:
maybe you can try commenting with // instead of /** */ since there is only one line of comment?

Reply:
I wasn't 100% sure as I noticed documentation for methods were usually written with multi-line comments, but I do agree that it would look cleaner with just `//`